cpu/nrf9160/Kconfig

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) 2021 Mesotic SAS
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+config CPU_FAM_NRF9160
+    bool
+    select CPU_COMMON_NRF5X
+    select HAS_CPU_NRF9160
+    select HAS_PERIPH_CPUID
+    select HAS_PERIPH_GPIO
+    select HAS_PERIPH_GPIO_IRQ
```

Comment:
This is already provided by `CPU_COMMON_NRF5X`

Reply:
But `cpu/nrf9160/Kconfig` doesn't inherit `cpu/nrf5x_common/Kconfig` so this is not the case.

Reply:
not sure if it has to, shouldn't it be enough that you select `CPU_COMMON_NRF5X` - @leandrolanzieri?

Reply:
I can remove the CPU_COMMON_NRF5X symbol for now, as long as this port is not fully align with nrf5x_common even if we use some drivers from it.

Reply:
I'm just surprised you can even select the symbol if you don't include it's definition.

Reply:
The symbol is not being selected, it is undefined, as @benpicco pointed out one needs to `source` the common file. I think there is a configuration to enable a warning when assigning a value to an undefined symbol, probably we need to enable that.

Reply:
I removed the symbol.